src/prototypes/single-article/example/example.twig

```diff
@@ -1,45 +1,4 @@
-<div class="o-page">
```

Comment:
This file's changes are unrelated to the Error pages, but it was only other spot using the Page template, so I updated it to match the better formatting I used on the Error pages, so there's less boilerplate, and it's easier to understand that this is using the Page object, the standard Sky Nav, and the standard Ground Nav.